tensilelite/Tensile/KernelWriter.py

```diff
@@ -978,7 +978,9 @@ def checkLocalReadFIFOFull(currentMFMA, fifo, lrItems, numLR, numLREven):
               break
           numToBeIssued += 1
         return numToBeIssued
-      
+
+      oneBufferScheduling = kernel["1LDSBuffer"] or kernel["DirectToLdsA"] or kernel["DirectToLdsB"]
```

Comment:
DirectToLds requires similar scheduling strategy as 1LDSBuffer.
We cannot overlap local read ad DTL global read (because DTL global read can overwrite LDS before read by local read).
Essentially ,we can manage it by using same scheduling logic as 1LDS buffer.